src/content/blog/fritzbox-routers-and-ddev.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Fritz!Box Routers and DDEV"
+pubDate: 2025-11-20
+summary: Solving DNS Rebinding issues with Fritz!Box routers (also spelled "Fritzbox" or "Fritz Box")
+featureImage:
+  src: /img/blog/2025/11/FritzBox-DNS-Rebind-Schutz.png
+  alt: Fritz!Box router network settings showing DNS rebinding protection options
+author: Randy Fay
+categories:
+  - TechNotes
+---
+
+DDEV is designed so that most people never have to change the configuration of their local workstation, and that includes not having to edit their hosts file. All the details are in [DNS Name Resolution and Wildcards](ddev-name-resolution-wildcards.md).
+
+However, one particular brand of router, the Fritz!Box, has a different DNS configuration than most other routers, and it includes DNS Rebinding Protection that blocks local development domains.
+
+**TL;DR:** _If you use a Fritz!Box router, add `ddev.site` to the router's DNS Rebinding Protection exceptions._
+
+## The Problem
+
+When you first set up DDEV with a Fritz!Box router, you might encounter a `DNS_PROBE_FINISHED_NXDOMAIN` error when trying to access your `.ddev.site` domain, even though your site is accessible via IP address. This happens because Fritz!Box routers include DNS Rebinding Protection that "suppresses DNS responses pointing to your own network."
```

Comment:
```suggestion
When you first set up DDEV with a Fritz!Box router, you might encounter a failure to resolve the domain name when trying to access your `*.ddev.site` project, even though your site is accessible via the `127.0.0.1` direct URL given in `ddev describe`. This happens because Fritz!Box routers enable DNS Rebinding Protection that suppresses DNS responses pointing to your own network.
```